package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "azuredatastudio",
   "version": "1.14.0",
-  "distro": "01d4471bbc2227c73907d5b42d6a54259e88b77c",
+  "distro": "08714f9db0e9c297aa44256ea74b708d5ec5ae05",
```

Comment:
For my own knowledge, what does this hash refer to?

Reply:
What commit to merge from.